LogicLayer/GameEvents/GameEventService.cs

```diff
@@ -37,7 +37,8 @@ public async Task CreateGameEventAsync(CreateGameEventDto requestedGameEvent)
                 Place = requestedGameEvent.Place,
                 Games = requestedGameEvent.Games
                     .Select((game, index) => new Game {Name = game, PositionOnTheList = index}).ToList(),
-                Participations = new List<GameEventParticipation>()
+                Participations = new List<GameEventParticipation>(),
+                Description = new DescriptionTab {Description = null}
```

Comment:
`Description = null` is redundant, `null` is the default.

Reply:
Done